projects/exui-common-lib/src/lib/components/generic-filter/generic-filter-utils.ts

```diff
@@ -0,0 +1,16 @@
+import {AbstractControl, ValidationErrors, ValidatorFn} from '@angular/forms';
+
+export function minSelectedValidator(min: number): ValidatorFn {
+  return (control: AbstractControl): ValidationErrors | null => {
+    const values = control.value as string[];
```

Comment:
Forcing this to be string[] restricts this to be used for a very specific type of selection. Better to make this a generic function (which can default to string[]):

export function minSelectedValidator<T = string>(min: number): ValidatorFn {
...
const values = control.value as T[];
...